packages/@react-spectrum/picker/stories/Picker.stories.tsx

```diff
@@ -531,14 +531,17 @@ function AsyncLoadingExample() {
 function ResizePicker() {
   const [state, setState] = useState(true);
 
-  return (<div>
-    <div style={{width: state ? '200px' : '300px'}}>
-      <Picker label="Choose A" UNSAFE_style={{width: '100%'}}>
-        <Item key="rarely">A1</Item>
-        <Item key="sometimes">A2</Item>
-        <Item key="always">A3</Item>
-      </Picker>
-    </div>
-    <ActionButton onPress={() => setState(!state)}>Toggle size</ActionButton>
-  </div>);
+  return (
+    <Flex direction="column" gap="size-200" alignItems="start">
+      <div>
+        <div style={{width: state ? '200px' : '300px'}}>
+          <Picker label="Choose A" UNSAFE_style={{width: '100%'}}>
+            <Item key="rarely">A1</Item>
+            <Item key="sometimes">A2</Item>
+            <Item key="always">A3</Item>
+          </Picker>
+        </div>
+        <ActionButton onPress={() => setState(!state)}>Toggle size</ActionButton>
+      </div>
+    </Flex>);
```

Comment:
Almost, the Flex won't do anything though with a single child. Also, as much as possible, we want to the use the style props instead of unsafe style.
```suggestion
  return (
    <Flex direction="column" gap="size-200" alignItems="start">
        <div style={{width: state ? '200px' : '300px'}}>
          <Picker label="Choose A" width="100%">
            <Item key="rarely">A1</Item>
            <Item key="sometimes">A2</Item>
            <Item key="always">A3</Item>
          </Picker>
        </div>
        <ActionButton onPress={() => setState(!state)}>Toggle size</ActionButton>
    </Flex>
  );
```